subprojects/platform-play/platform-play.gradle

```diff
@@ -49,8 +49,8 @@ task integTestPrepare(type: org.gradle.gradlebuild.test.integrationtests.Integra
     maxParallelForks = 1
 }
 
-tasks.configureEachLater(org.gradle.gradlebuild.test.integrationtests.IntegrationTest) { task ->
-    if (task != integTestPrepare) {
+tasks.withType(org.gradle.gradlebuild.test.integrationtests.IntegrationTest).configureEach { task ->
+    if (task.name != "integTestPrepare") {
```

Comment:
Could we be more robust here and use `task.name != integTestPrepare.name`? Actually, looks like `TaskProvider` doesn't implement `Named`.

Reply:
yeah, and the obvious thing also doesn't work (task is a Task and integTestPrepare is a Provider):

```
if (task != integTestPrepare) {
...
}
```